lib/services/large_result_set.js

```diff
@@ -70,6 +70,10 @@ function LargeResultSetService(connectionConfig, httpClient)
       // err happens on timeouts and response is passed when server responded
       if (err || isUnsuccessfulResponse(response))
       {
+        // if we're running in DEBUG loglevel, probably we want to see the full error too
+        const logErr = err ? JSON.stringify(err, Util.getCircularReplacer()) 
+        : JSON.stringify(response, Util.getCircularReplacer());
```

Comment:
Not having bigger picture - but this is potentially too much? Are we sure, that no sensitive data is logged?

Reply:
we already log the response headers even for a [successful response](https://github.com/snowflakedb/snowflake-connector-nodejs/blob/v1.9.1/lib/services/large_result_set.js#L104-L106) but definitely can use similar approach here too

Reply:
done